crates/fluvio-spu-schema/src/produce/request.rs

```diff
@@ -46,8 +53,8 @@ where
     const API_KEY: u16 = 0;
 
     const MIN_API_VERSION: i16 = 0;
-    const MAX_API_VERSION: i16 = 7;
-    const DEFAULT_API_VERSION: i16 = 7;
+    const MAX_API_VERSION: i16 = 8;
```

Comment:
It's better to use here the newly introduced const `PRODUCER_TRANSFORMATION_API_VERSION` to prevent mistakes in the future.